validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -73,6 +69,7 @@
   private static final String SCHEMA_SUFFIX = ".json";
   private static final String REGISTRATION_SUMMARY_JSON = "registration_summary.json";
   private static final String SCHEMA_NAME = "UDMI";
+  private static final String SITE_DEFAULTS_JSON = "site_defaults.json";
```

Comment:
I was pondering if this might be better named "metadata_defaults.json" -- just because there might be other defaults that apply to the site that aren't metadata... I don't have a specific alternative, but trying to future-proof the name. (The sites part is also implied by it being at the top-level.) OTOH, site_defaults.json makes the name itself unique in the case when there are other kinds of default metadata (e.g. device type).

Reply:
I would agree I'm not a fan of the name now that we use the Metadata schema instead of a different, related one. It doesn't show the relationship. As long as the naming allows room for other defaults to named by a similar method I'm ok with it. metadata_defaults, then?

Reply:
Another name to consider would be "site_metadata.json" which would (eventually then) be unique compared with a "hardware_metadata.json" (e.g. for all Delta O3 devices) or "entity_metadata.json" (that would be valid for, e.g. all "FCU"s in the system regardless of the hardware).  I'm thinking that's better than "metadata_defaults.json" which is still ambiguous (is it the default for the { site, entity, hardware }?)...  Personally, after this thinking I'd be leaning towards "site_metadata.json"... ?

Reply:
Ok so site_metadata?

Reply:
SGTM